<?xml version="1.0" ?>
<launch>
    <rosparam file = "$(find challenge1)/config/path.yaml" command = "load"  />
    <node name="script_selector" pkg="challenge1" type="script_selector.py" output="screen"/>
    <node name="path_follower" pkg="challenge1" type="path_follower.py" output="screen"/>
    <node name="rqt_graph" pkg="rqt_graph" type="rqt_graph" output="screen" />
</launch>
